src/input/LibretroDevice.cpp

```diff
@@ -90,7 +100,21 @@ bool CLibretroDevice::Deserialize(const TiXmlElement* pElement)
       return false;
     }
 
-    m_featureMap[name] = mapto;
+    std::string libretroFeature;
+
+    if (buttonMapVersion == 1)
+      libretroFeature = LibretroTranslator::GetFeatureV2(mapto);
```

Comment:
Hm, seems like GetFeatureV2 only works for game controllers, but it can't be used for mouse as it doesn't differentiate controller types. Mouse has features "left" and "right" that would require a different mapping.

As far as I know there are currently only 2 cores that use the mouse support (scummvm, dosbox), so I've just migrated those 2 buttonmaps to v2: https://github.com/kodi-game/game.libretro.scummvm/pull/11, https://github.com/kodi-game/game.libretro.dosbox/pull/3.